docs/content/doc/help/faq.en-us.md

```diff
@@ -160,18 +161,26 @@ Which makes all other paths non-writeable to Gitea.
 Our translations are currently crowd-sourced on our [Crowdin project](https://crowdin.com/project/gitea)  
 Whether you want to change a translation or add a new one, it will need to be there as all translations are overwritten in our CI via the Crowdin integration.
 
-## SSH issues
+## Hooks aren't running
+If Gitea is not running hooks, a common cause is incorrect setup of SSH keys.  
+See [SSH Issues](#ssh-issues) for more information.  
+  
+You can also try logging into the administration panel and running the `Resynchronize pre-receive, update and post-receive hooks of all repositories.` option.
 
+## SSH issues
 If you cannot reach repositories over `ssh`, but `https` works fine, consider looking into the following.
 
 First, make sure you can access Gitea via SSH.  
 `ssh -i ~/.ssh/<keyfile> git@myremote.example`  
```

Comment:
Should this example just be a normal ssh command ```ssh git@remote.example``` and not specify the key file? Perhaps a common error is that people create a key that doesn't have a default name like ```id_rsa``` and then it is not used when running a normal ssh command like ```git clone ssh://git@myremote.example``` because they don't have something like this in their config file :
```
Host myremote.example
  IdentityFile ~/.ssh/id_rsa_gitea
```
The test above might mask that problem since it would work in that case of specifying the keyfile but a normal clone command will not if they haven't set up their config to use that key.

Reply:
That's a good point, done.